hugo/layouts/products/community-edition/single.html

```diff
@@ -23,7 +23,7 @@ <h3 class="text-medium-light mb-4">Built in UI and integrations with your favori
                 <p class="text-dark">With source{d} you can start getting answers to tough questions in a matter of minutes with simple SQL queries. source{d} supports all major operating systems (Linux, Windows, MacOS), hosting environments (Cloud or On-premise), version control systems (GitHub, GitLab, Bitbucket, etc) and adapts to different stages of development, workflows and business needs. Enjoy the flexibility to choose whether to work on our built-in interactive web interface, your favorite SQL client, command-line interface or Business Intelligence platform (Tableau, Looker, PowerBI, etc).</p>
                 <div class="text-center mt-5 row">
                     <div class="col-xl-6">
-                        <a href="https://go.sourced.tech/engine-download" class="btn btn-purple text-white mb-4 w-100">Download source{d}</a>
+                        <a href="https://go.sourced.tech/community-edition-download" class="btn btn-purple text-white mb-4 w-100">Download source{d}</a>
                     </div>
                     <div class="col-xl-6">
                         <a href="https://go.sourced.tech/engine" class="btn btn-highlight w-100">Request a Demo</a>
```

Comment:
same as in https://github.com/src-d/landing/pull/441/files#diff-04c805a4a1460b51fa27d4bd4e90b327R57
```suggestion
                        <a href="https://go.sourced.tech/community-demo" class="btn btn-highlight w-100">Request a Demo</a>
```

Reply:
Fixed!